File: core/src/main/java/com/farsitel/bazaar/core/callback/BazaarSignInCallback.java
package com.farsitel.bazaar.core.callback;

import com.farsitel.bazaar.BazaarResponse;
import com.farsitel.bazaar.core.model.BazaarSignInAccount;

public interface BazaarSignInCallback {
    void onAccountReceived(BazaarResponse<BazaarSignInAccount> account);
}

File: core/src/main/java/com/farsitel/bazaar/core/callback/BazaarSignInCallback.kt
package com.farsitel.bazaar.core.callback

import com.farsitel.bazaar.BazaarResponse
import com.farsitel.bazaar.core.model.BazaarSignInAccount

fun interface BazaarSignInCallback {
    fun onAccountReceived(account: BazaarResponse<BazaarSignInAccount>)
}